modules/test/playwright/tests/object-web/objectClientExtensions.spec.ts

```diff
@@ -168,3 +198,82 @@ test('Can create, read, update, and delete object entries that use the client ex
 			.getByText(objectEntryUpdatedValue, {exact: true})
 	).toBeHidden();
 });
+
+test('Can trigger object validation as a client extension', async ({
+	editObjectValidationPage,
+	modalAddObjectValidationPage,
+	objectValidationsPage,
+	page,
+	viewObjectEntriesPage,
+}) => {
+	const [objectDefinition] = createdEntities.objectDefinitions;
+
+	const objectField = objectDefinition.objectFields.find(
+		({system}) => !system
+	);
+
+	await objectValidationsPage.goto(objectDefinition.label['en_US']);
+
+	await objectValidationsPage.addObjectValidationButton.click();
+
+	const objectValidationLabel = 'ClientExtensionValidation' + getRandomInt();
+
+	await modalAddObjectValidationPage.fillObjectValidationInputs(
+		objectValidationLabel,
+		'Liferay Sample Etc Spring Boot Spring Boot Object Validation Rule 1'
+	);
+
+	await page.getByText(objectValidationLabel).click();
+
+	await objectValidationsPage.activeValitionToggle.check();
+
+	await objectValidationsPage.errorMessageInput.fill(
+		'This entry is not possible.'
+	);
+
+	await editObjectValidationPage.saveObjectValidationButton.click();
+
+	await page.waitForEvent('domcontentloaded');
+
+	await viewObjectEntriesPage.goto(objectDefinition.id);
+
+	await viewObjectEntriesPage.clickAddObjectEntry(
+		objectDefinition.label['en_US']
+	);
+
+	await viewObjectEntriesPage.fillObjectEntry({
+		objectFieldBusinessType: objectField.businessType,
+		objectFieldLabel: objectField.label['en_US'],
+		objectFieldValue: 'Invalid Name',
+	});
+
+	await viewObjectEntriesPage.saveObjectEntryButton.click();
+
+	await waitForAlert(page, 'This entry is not possible.', {type: 'danger'});
+
+	await viewObjectEntriesPage.backButton.click();
+
+	await expect(
+		page.locator('.cell-name').getByText('Invalid Name')
+	).toBeHidden();
```

Comment:
May be `not.toBeAttached` is a better option, not sure, because if there is a hidden occurance of the element, this will evaluate to true.

Check discussion [here](https://stackoverflow.com/questions/78003346/how-can-i-distinguish-between-element-not-visible-and-not-rendered).

And docs for `toBeAttached` [here](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-attached).

Reply:
toBeHidden is described like

> Ensures that [Locator](https://playwright.dev/docs/api/class-locator) either does not resolve to any DOM node, or resolves to a [non-visible](https://playwright.dev/docs/actionability#visible) one.

Reply:
Within the context of the test, I believe that both functions would fit here, but I am making the change 😄

Reply:
Furthermore, the `toBeAttached` function has a parameter {attached: boolean}, which eliminates the need to use .not.toBeAttached()